README.md

```diff
@@ -11,18 +11,7 @@ This module is a dependency of the agent and is installed by default when you in
 
 ## Installation
 
-Typically, most users use the version auto-installed by the agent. You can see agent install instructions [here](https://github.com/newrelic/node-newrelic#installation-and-getting-started).
-
-In some cases, installing a specific version is ideal. For example, new features or major changes might be released via a major version update to this module, prior to inclusion in the main New Relic Node.js Agent.
-
-```
-npm install @newrelic/koa
-```
-
-```js
-// index.js
-require('@newrelic/koa')
-```
+This package is [a dependency of the the Node Agent](https://github.com/newrelic/node-newrelic/blob/main/package.json#L189), and the average user should not need to install it manually.
```

Comment:
Do we want to link to the package.json here? Or is the ReadMe enough?

Reply:
That's the language in the other external repos so I just wanted to stay consistent